openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/NodeSuggestions.component.tsx

```diff
@@ -50,16 +50,16 @@ const NodeSuggestions: FC<EntitySuggestionProps> = ({
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const [searchValue, setSearchValue] = useState<string>('');
 
-  const getSuggestionLabel = (fqn: string, type: string, name: string) => {
+  const getSuggestionLabelHeading = (fqn: string, type: string) => {
     if (type === EntityType.TABLE) {
       const database = getPartialNameFromTableFQN(fqn, [FqnPart.Database]);
       const schema = getPartialNameFromTableFQN(fqn, [FqnPart.Schema]);
 
       return database && schema
-        ? `${database}${FQN_SEPARATOR_CHAR}${schema}${FQN_SEPARATOR_CHAR}${name}`
-        : name;
+        ? `${database}${FQN_SEPARATOR_CHAR}${schema}`
```

Comment:
Need to verify this, We will only show it for database & schema

Reply:
Verified on existing implementation. We only show for database & schema.